web/styles/left_sidebar.css

```diff
@@ -540,6 +540,72 @@ li.topic-list-item:hover .topic-sidebar-menu-icon {
     color: hsl(0deg 0% 53%);
 }
 
+.top-left-sidebar-menu-icon {
+    display: none;
+}
+
+.icon_layout {
```

Comment:
This CSS class name is too generic -- there's lots of things in the app that it could potentially refer to. When naming elements, especially those that will have top-level CSS rules like this one, try to pick something unambiguous like `top-left-layout-icons`.